### PR TITLE
Update k8scc and update AMIs

### PR DESCRIPTION
**Pull request by tfussell**
Updating k8scc with https://github.com/giantswarm/k8scloudconfig/pull/783 for https://github.com/giantswarm/giantswarm/issues/12619.

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Comment by tfussell** [comment]
Tested on `gauss`. Creation and upgrades are good. No notable errors in operator logs.

File: go.mod [hunk 1]
```diff
@@ -19,7 +19,7 @@ require (
 	github.com/giantswarm/helmclient v0.2.0
 	github.com/giantswarm/ipam v0.2.0
 	github.com/giantswarm/k8sclient v0.2.0
-	github.com/giantswarm/k8scloudconfig/v6 v6.1.2
+	github.com/giantswarm/k8scloudconfig/v6 v6.1.3-0.20200827212125-e3aa902e48b2
```
Comment (tfussell) [inline go.mod:22]: Will update to v6.1.3 when I merge https://github.com/giantswarm/k8scloudconfig/pull/784 tomorrow.

**Comment by tfussell** [comment]
FYI @giantswarm/team-firecracker

**Review by MarcelMue — APPROVED** [review]

**Review by calvix — APPROVED** [review]

File: service/controller/key/zz_generated.ami.go [hunk 2]
```diff
@@ -210,6 +210,28 @@ var amiJSON = []byte(`{
     "us-east-2": "ami-083c654d2f0469f9e",
     "us-west-1": "ami-03efdc564552f8ed5",
     "us-west-2": "ami-0bb54692374ac10a7"
+  },
+  "2512.3.0": {
```
Comment (njuettner) [inline service/controller/key/zz_generated.ami.go:214]: How is this generated?

**Review by njuettner — APPROVED** [review]

File: service/controller/key/zz_generated.ami.go (hunk 2 above)
Comment (MarcelMue) [reply service/controller/key/zz_generated.ami.go:214]: Here is the docs on it: https://github.com/giantswarm/aws-operator#node-vm-images-amis
`devctl gen ami --dir service/controller/key` basically does the job :)

Comment (njuettner) [reply service/controller/key/zz_generated.ami.go:214]: Nice, didn't saw it. Thanks :)

**Pull request merged**